app/src/main/java/space/taran/arknavigator/utils/FileUtils.kt

```diff
@@ -48,6 +63,9 @@ fun isHidden(path: Path): Boolean =
 fun folderExists(path: Path): Boolean =
     Files.exists(path) || Files.isDirectory(path)
 
+fun isFileEmpty(filePath: Path?): Boolean =
```

Comment:
I think this function is little bit harmful. We should not tolerate null paths, if we encounter them in `GalleryFragment` previews, then it is a mistake and we have to throw `AssertionError()`.